### ISSUE TITLE
nouns with á as the last vowel

### ISSUE THREAD

**Issue by jonorthwash** [apertium/apertium-kaa#7]
There seems to be something slightly unexpected with vowel harmony related to stems where the last vowel is á.

For example, I have seen forms like dúkánnan, but also gápten.  So the question is: after á, how do we know whether we get a front or back vowel?  I'm having trouble working this out on my own, because stems with the last vowel á are somewhat uncommon.

So which of the following forms are correct, and which are not?

| form | back form | front form | 
|---|---|---|
| `acc` | dúkándı, gáptı | dúkándi, gápti |
| `gen` | dúkánnıń, gáptıń | dúkánniń, gáptiń |
| `loc` | dúkánda, gápta | dúkánde, gápte |
| `abl` | dúkánnan, gáptan | dúkánnen, gápten |
| `dat` | dúkánǵa, gápqa | dúkánge, gápke |
| `ins` | dúkánman, gáppan | dúkánmen, gáppen |
| `px3sp.nom` | dúkánı, gábı | dúkáni, gábi |

(Note that there are other hypothetical variants for `dat`, like dúkánga and dúkánǵe, but I figure these aren't correct.)

@MambetniyazovAmir, could you
- indicate which of these forms are okay, and which are not okay? 
- also give correct case forms (and a possessive form) for some other nouns, like shárt, jámáát, báhár, jáhán, dán, zúlqádá?

**Comment by MambetniyazovAmir** [comment]
1.  front form is correct
2. 
shárttıń 	jámááttıń 	báhárdiń 	        jáhánniń 	        dánniń 	
shártge 	jámáátge 	báhárge 	        jáhánge 	        dánge 	
shártti 	jámáátti 	        báhárdi 	        jáhándi 	        dándi 	
shártten 	jámáátten 	báhárden 	jáhánnen 	dánnen 	
shártte 	jámáátte 	        báhárde 	        jáhánde 	        dánde

**Comment by jonorthwash** [comment]
> 1. front form is correct

But what about forms like dúkánnan?  Is that incorrect?

> shártge jámáátge báhárge jáhánge dánge

shártke and jámáátke, right?

**Comment by MambetniyazovAmir** [comment]
1. Yes, it is incorrect.
2. yes, shártke and jámáátke are correct

**Comment by jonorthwash** [comment]
What about dúkánnan in this context?  Is it wrong?

![83810931-9829da00-a6c1-11ea-9941-0f613cc83489](https://user-images.githubusercontent.com/963849/84557571-6d133c00-acfa-11ea-99a1-c5a69b969ce2.png)

**Comment by MambetniyazovAmir** [comment]
No, it isn't wrong. you are right. it will be dúkánnan. qatelesippen
